src/pages/plugins/index.hbs

```diff
@@ -3,6 +3,8 @@ class: plugins
 ---
 
 <div class="container">
+
+  <h3>Official</h3>
```

Comment:
```suggestion
  <h3>Official</h3>

  <p>
    Open-source plugins maintained and managed by Esri.
  </p>

```

Here's an idea for a "subtitle" of sorts to explain what is meant by "Official"; however, I really don't want this language to overlook all the help Esri Leaflet and "official" plugins have had from non-Esri-employee contributors.  Is there another way we can phrase my suggestion here?